app/src/main/res/values/strings.xml

```diff
@@ -211,6 +211,7 @@
   <string name="nearby_location_has_not_changed">Location has not changed.</string>
   <string name="nearby_location_not_available">Location not available.</string>
   <string name="location_permission_rationale_nearby">Permission required to display a list of nearby places</string>
-    <string name="get_directions">GET DIRECTIONS</string>
+  <string name="get_directions">GET DIRECTIONS</string>
   <string name="read_article">READ ARTICLE</string>
+  <string name="toggle_view_button">Toggle views</string>
```

Comment:
Actually, would "Toggle view" sound better for this? I'd like to have a native speaker's opinion.

Reply:
Can a native speaker answer this so that I can merge this :)? @misaochan @nicolas-raoul

Reply:
Unfortunately I cannot claim to be a native speaker per se :) , but I would go with "toggle view" as that appears to be the more common usage.

Reply:
Thanks, now changed to "Toggle view".